test/evergreen.yml

```diff
@@ -544,7 +555,7 @@ functions:
             exit 0
           fi
 
-          git clone git@github.com:wiredtiger/wiredtiger.github.com.git
+          git clone https://github.com/wiredtiger/wiredtiger.github.com.git
```

Comment:
`wiredtiger/wiredtiger.github.com` is a public repo so https clone should work. The following `shell.exec` command has already taken care of the token generation from github app (in a more manual way than the `github.generate_token` Evergreen command).